proto/api.proto

```diff
@@ -660,18 +662,30 @@ message GetQuotesResponse{
 }
 
 message ProjectQuote {
-  string project = 1;
+  Project project = 1;
   repeated QuoteRoute routes = 2;
 }
 
+enum Project {
+  P_UNKNOWN = 0;
+  P_JUPITER = 1;
+  P_RAYDIUM = 2;
+  P_ALL = 3;
+}
+
 message QuoteRoute {
-  string inToken = 1;
-  string inTokenAddress = 2;
-  string outToken = 3;
-  string outTokenAddress = 4;
-  double inAmount = 5;
-  double outAmount = 6;
-  double slippage = 7;
-  double priceImpact = 8;
-  double fee = 9;
+  repeated QuoteStep steps = 1;
+}
+
+message QuoteStep {
+  string exchange = 1;
```

Comment:
might need to think a bit about this format and particularly "project" (a few levels up) and "exchange" here. I understand the need for it to be like this now, just not sure exchange is the word for it. Might need to reuse "project". We'll talk with Daniel. For now it should do it